locations/saarbruecker.yml

```diff
@@ -1,17 +1,34 @@
 ---
-
 location: saarbruecker
 location_nice: Saarbrücker
 latitude: 52.5283930134242
 longitude: 13.41561550087894
 community: true
 
 hosts:
-
   - hostname: saarbruecker-gw
     role: gateway
     model: "ubnt_edgerouter-4"
 
+snmp_devices:
+  - hostname: saarbruecker-sw
+    address: 10.31.83.50
+    snmp_profile: edgerouter
+
+  # hds has no roof installation as of 02/24
+  # - hostname: saarbruecker-hds
+  #   address: 10.31.83.51
+  #   snmp_profile: af60
```

Comment:
this device is up and running, even if it has no neighbor it should be included. But it's MikroTik, not AF60